charts/core/templates/_sentry.tpl

```diff
@@ -0,0 +1,41 @@
+{{/**
+  config:
+    <...>
+    sentry:
+      enabled: false
+      # -- Sentry DSN
+      dsn: ""
+      # -- Sentry environment
+      environment: ""
+      # -- Sentry release
+      release: ""
+      # -- Sentry Auth Token
+      authToken:
+        value: ""
+        existingSecret: ""
+        secretKeys:
+          value: ""
+    <...>
+**/}}
+
+{{- define "core.sentry" }}
+- name: SENTRY
+  value: {{ .Values.config.sentry.enabled | quote }}
+{{- if .Values.config.sentry.enabled }}
+- name: SENTRY_DSN
+  value: {{ .Values.config.sentry.dsn }}
+- name: SENTRY_ENVIRONMENT
+  value: {{ .Values.config.sentry.environment }}
+- name: SENTRY_RELEASE
+  value: {{ .Values.config.sentry.release }}
+- name: SENTRY_AUTH_TOKEN
+  {{- if gt (len .Values.config.sentry.authToken.existingSecret) 0 }}
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.config.sentry.authToken.existingSecret }}
+      key: {{ .Values.config.sentry.authToken.secretKeys.value }}
+  {{- else }}
+  value: {{ .Values.config.sentry.authToken.value }}
+  {{- end }}
+{{- end -}}
+{{- end -}}
```

Comment:
cc @reslene
This is now common for console, console-v3, portal. 

If something does not match there is still `.config.additionnalEnv`. There is a trick to actually `remap`

If the service is not ready:
```
- name: "SENTRY_AUTH_TOKEN_2"
  value: "$SENTRY_AUTH_TOKEN"
```